drf_dx_datagrid/viewsets.py

```diff
@@ -10,6 +10,7 @@
 
 class DxModelViewSet(rest_framework.viewsets.ModelViewSet, DxMixin, SummaryMixin):
     pagination_class = TakeSkipPagination
+    is_case_sensitive = True
```

Comment:
This row will change default behaviour after package updating. It's not good. It may be worth adding this code in filters.py :

def get_case_sesitive():
   from django.conf import settings
    try:
        return settings.REST_FRAMEWORK['DRF_DX_DATAGRID']['FILTER_CASE_SENSITIVE']
    except (AttributeError, KeyError):
        return False

In settings.py:
REST_FRAMEWORK = {
'DRF_DX_DATAGRID': {
'FILTER_CASE_SENSITIVE': True}
}

Reply:
>This row will change default behaviour after package updating.
It's not. Default behavior is case sensitive.
To switch to insensitive filtering (not default) endpoint viewset have to change it to False.

Settings is pretty good, we cannot define different sensitivity for viewsets, but it is not needed in my particular case, and i guess in most cases ever.